code/src/test/kotlin/com/expediagroup/sdk/core/authentication/bearer/TokenResponseTest.kt

```diff
@@ -0,0 +1,129 @@
+package com.expediagroup.sdk.core.authentication.bearer
+
+import com.expediagroup.sdk.core.http.CommonMediaTypes
+import com.expediagroup.sdk.core.http.Method
+import com.expediagroup.sdk.core.http.Protocol
+import com.expediagroup.sdk.core.http.Request
+import com.expediagroup.sdk.core.http.Response
+import com.expediagroup.sdk.core.http.ResponseBody
+import com.expediagroup.sdk.core.http.Status
+import com.expediagroup.sdk.core.model.exception.client.ExpediaGroupResponseParsingException
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+
+class TokenResponseTest {
+    @Test
+    fun `parsed instances accessToken value should map to api response`() {
```

Comment:
```suggestion
    fun `should map to the expected api response`() {
```
We can then combine this test with the one below. Just add the assertion for the `expiresIn`

Reply:
done.